packages/xrpl/test/models/XChainAddClaimAttestation.test.ts

```diff
@@ -0,0 +1,334 @@
+      ValidationError,
+      'XChainAddClaimAttestation: missing field PublicKey',
+    )
+  })
+
+  it('throws w/ invalid PublicKey', function () {
+    tx.PublicKey = 123
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field PublicKey',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field PublicKey',
+    )
+  })
+
+  it('throws w/ missing Signature', function () {
+    delete tx.Signature
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field Signature',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field Signature',
+    )
+  })
+
+  it('throws w/ invalid Signature', function () {
+    tx.Signature = 123
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field Signature',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field Signature',
+    )
+  })
+
+  it('throws w/ missing WasLockingChainSend', function () {
+    delete tx.WasLockingChainSend
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field WasLockingChainSend',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field WasLockingChainSend',
+    )
+  })
+
+  it('throws w/ invalid WasLockingChainSend', function () {
+    tx.WasLockingChainSend = 2
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field WasLockingChainSend',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field WasLockingChainSend',
+    )
+  })
+
+  it('throws w/ missing XChainBridge', function () {
+    delete tx.XChainBridge
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field XChainBridge',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: missing field XChainBridge',
+    )
+  })
+
+  it('throws w/ invalid XChainBridge', function () {
+    tx.XChainBridge = { XChainDoor: 'test' }
+
+    assert.throws(
+      () => validateXChainAddClaimAttestation(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field XChainBridge',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'XChainAddClaimAttestation: invalid field XChainBridge',
+    )
+  })
+
+  it('throws w/ missing XChainClaimID', function () {
```

Comment:
nit: I think the `w/` shorthand is less readable than `with` for next to no simplicity/length gain. Should apply throughout
```suggestion
  it('throws with missing XChainClaimID', function () {
```

Reply:
It looks like most tests use `w/` instead of `with`, so I'd rather keep it this way for the sake of uniformity. It'd be a pretty simple Ctrl-F-Replace if you wanted to replace them all in a separate PR, though.